pkg/server/server.go

```diff
@@ -122,6 +115,7 @@ func (s *Server) ListObjects(ctx context.Context, req *openfgapb.ListObjectsRequ
 		ListObjectsMaxResults: s.config.ListObjectsMaxResults,
 		ResolveNodeLimit:      s.config.ResolveNodeLimit,
 		CheckResolver:         s.checkResolver,
+		Typesystem:            typesys,
```

Comment:
this is breaking the pattern we currently have where the `command` object has fields that are not request-specific. `typesys` is request specific

Reply:
I'm not sure I understand. The `ListObjectsQuery` itself is request-bound. The query is constructed per-request, and it serves the inbound `ListObjects` RPC that is being invoked. Consequently, the Typesystem is very applicable to the incoming request. If we only constructed a single `ListObjectsQuery` up-front and then called `Execute` per invocation, then I could understand the point, but that's not the case here.

I was under the impression this was the direction you preferred our code-base trended towards, so this comment is surprising to me? We've been passing Typesystem through context, and you have been arguing in favor of passing Typesystem explicitly to the `commands`.

Reply:
Sorry i didn't explain myself super well. If you look at the codebase today, at our commands, e.g.

https://github.com/openfga/openfga/blob/5185d4e041e8b8ad4692a34e912a8bc891e465b1/pkg/server/commands/read_authzmodel.go#L14-L17

or 

https://github.com/openfga/openfga/blob/5185d4e041e8b8ad4692a34e912a8bc891e465b1/pkg/server/commands/read_changes.go#L15-L20

the fields in the `Query` struct (commands) are global objects that do not vary per request. Today, if we want to, we could build one of these objects at the server constructor and re-use it per request.

But in this PR, you are constructing `typesys` _from the request parameters_ and adding it to this struct. This would make us not be able to do what I mentioned above